projects/alfresco/nginx/templates/nginx.conf

```diff
@@ -0,0 +1,79 @@
+upstream tomcat_server {
+        server TOMCAT_REPLACE_ME max_fails=3 fail_timeout=10s;
+}
+upstream kibana {
+        server KIBANA_REPLACE_ME max_fails=3 fail_timeout=10s;
+}
+server {
+        server_tokens off;
+        listen 80;
+        client_max_body_size 0;
+        client_body_buffer_size 128K;
+        proxy_read_timeout 600s;
+        server_name spg-alf-310;
+     
+        location / {
+                rewrite ^/$ /share;
+        }
+     
+        location /share {
+                proxy_pass http://tomcat_server/share;
+                proxy_set_header        X-Real-IP       $remote_addr;
+                proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
+                proxy_set_header        host    $host;
+                proxy_set_header        X-Forwarded-Server      $host;
+        }
+     
+        location ~ (^/share/.*/thumbnails/.*$)  {
+                proxy_pass http://tomcat_server;
+                proxy_set_header        X-Real-IP       $remote_addr;
+                proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
+                proxy_set_header        host    $host;
+                proxy_set_header        X-Forwarded-Server      $host;
+        }
+     
+        location /alfresco {
+                proxy_pass http://tomcat_server/alfresco;
+                proxy_set_header        X-Real-IP       $remote_addr;
+                proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
+                proxy_set_header        host    $host;
+                proxy_set_header        X-Forwarded-Server      $host;
+        }
+ 
+        location /noms-spg {
+                proxy_pass http://tomcat_server/alfresco/service/noms-spg;
+                proxy_set_header        X-Real-IP       $remote_addr;
+                proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
+                proxy_set_header        host    $host;
+                proxy_set_header        X-Forwarded-Server      $host;
+        }
+ 
+        location /admin-spg {
+                proxy_pass http://tomcat_server/alfresco/service/admin-spg;
+                proxy_set_header        X-Real-IP       $remote_addr;
+                proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
+                proxy_set_header        host    $host;
+                proxy_set_header        X-Forwarded-Server      $host;
+ 
+        }
+  
+        location /mts-spg {
+                proxy_pass http://tomcat_server/alfresco/service/mts-spg;
+                proxy_set_header        X-Real-IP       $remote_addr;
+                proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
+                proxy_set_header        host    $host;
+                proxy_set_header        X-Forwarded-Server      $host;
+        }
+ 
+        location /kibana {
```

Comment:
kibana lives on `monitoring.<target>.internal:5601` or `monitoring.<target>.probation.hmpps.dsd.io` we'll need to make sure this rewrite works correctly